tests/dummy/config/deprecation-workflow.js

```diff
@@ -0,0 +1,20 @@
+/* global window */
+
+window.deprecationWorkflow = window.deprecationWorkflow || {};
+window.deprecationWorkflow.config = {
+  workflow: [
+    {
+      handler: 'silence',
+      matchId: 'ember-polaris.polaris-description-list.term-component',
+    },
+    {
+      handler: 'silence',
+      matchId: 'ember-polaris.polaris-description-list.description-component',
+    },
+    { handler: 'silence', matchId: 'computed-property.override' },
+    {
+      handler: 'silence',
+      matchId: 'ember-polaris.polaris-labelled.dataTestLabelled-arg',
+    },
+  ],
+};
```

Comment:
Curious, why do we want to silence these in the dummy app? 🤔

Reply:
to reduce the noise while working through these PRs